src/lottery/services/transactions.js

```diff
@@ -13,7 +20,9 @@ const getUserTransactionsHandler = async (req, res) => {
       .lean();
     if (transactions)
       res.json({
-        transactions,
+        transactions: transactions.map((transaction) =>
+          hideItemStock(transaction)
+        ),
```

Comment:
`f = (x) => g(x)`는 `g`랑 동일합니다.

```suggestion
        transactions: transactions.map(hideItemStock)
```

Reply:
아 맞네요! 감사합니다. 다음 커밋에서 수정할게요!